royaloperahouse/stream-tv-app
# #76 added supporting of exiting from app; Android only

**What?**
#76 Warning when exiting the app
added supporting of exit from the app

**Why?**
It is able users to choose another Firestick app. 

**How?**

added additional item(button) to nav menu. added modal with exit`s question.

works only on Android devices. For iOS need to find another approach
https://github.com/wumke/react-native-exit-app  as example 

**How to test?**

press on the exit button at bottom of the nav menu. you will see the modal screen with the exit`s question. If you will agree with it the app will clear all states and close. It will be equal if you press the back button on the remote control in the main screens as home screen, search screen, and so on.
